src/Tickets/Commerce/Currency.php

```diff
@@ -11,4 +11,41 @@
  */
 class Currency {
 
+	/**
+	 * The option key that stores the currency code in tribe options
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	public static $currency_code_option = 'tickets-commerce-currency-code';
+
+	/**
+	 * Retrieves the working currency set in Tribe Commerce or Tickets Commerce
+	 *
+	 * @todo  @backend: due to time constraints, not all calls to tribe_get_option( 'ticket-commerce-currency-code' )
+	 *       were replaced by this method. That key had a typo in it, and we're fixing that now. We need to clean up
+	 *       and have the entire codebase read currency codes from this class in the future. Once all currency
+	 *       operations are fixed to use this class, modify this method to delete the old keys from the settings array.
+	 *
+	 * @since TBD
+	 *
+	 * @return string
+	 */
+
+	public static function get_currency_code() {
+		// New key.
+		$currency = tribe_get_option( static::$currency_code_option );
+
+		if ( empty( $currency ) ) {
+			// Old key.
+			$currency = tribe_get_option( 'ticket-commerce-currency-code', 'USD' );
```

Comment:
Comment here should make reference to Tribe Commerce specifically, not just old way.
Also before doing the update to a empty check on the value in case it's set as 0 or '', which will not return the USD value for default. Old code that is behind `tribe_get_option` being weird.

Reply:
Not sure I understand what you mean. This piece is here to say "if the proper TC option is not set, get the TPP option, annd if that is also not set, fall back to USD". This allows us to change `Tribe/Commerce/Currrency.php` keeping the TPP and TC functionalities working together.

With ET-1331 we should probably move away from `Tribe/Commerce/Currency.php` entirely, but I guess that's off the table for this release.